api.php

```diff
@@ -662,13 +694,143 @@
+     * @apiSuccess {boolean} data.sanity_running Whether the sanity process is currently running.
+     * @apiSuccess {number}  data.last_sanity The timestamp for the last time the sanity process was run.
+     * @apiSuccess {boolean} data.sanity_sync Whether the sanity process is currently synchronising.
+     */
+    $sanity = file_exists(__DIR__.'/tmp/sanity-lock');
+
+    $lastSanity = (int)$db->single("SELECT val FROM config WHERE cfg='sanity_last'");
+    $sanitySync = (bool)$db->single("SELECT val FROM config WHERE cfg='sanity_sync'");
+    api_echo(['sanity_running' => $sanity, 'last_sanity' => $lastSanity, 'sanity_sync' => $sanitySync]);
+} elseif ($q === 'node-info') {
+    /**
+     * @api            {get} /api.php?q=node-info  21. node-info
+     * @apiName        node-info
+     * @apiGroup       API
+     * @apiDescription Returns details about the node.
+     *
+     * @apiSuccess {object}  data A collection of data about the node.
+     * @apiSuccess {string} data.hostname The hostname of the node.
+     * @apiSuccess {string} data.version The current version of the node.
+     * @apiSuccess {string} data.dbversion The database schema version for the node.
+     * @apiSuccess {number} data.accounts The number of accounts known by the node.
+     * @apiSuccess {number} data.transactions The number of transactions known by the node.
+     * @apiSuccess {number} data.mempool The number of transactions in the mempool.
+     * @apiSuccess {number} data.masternodes The number of masternodes known by the node.
+     * @apiSuccess {number} data.peers The number of valid peers.
+     */
+    $dbVersion = $db->single("SELECT val FROM config WHERE cfg='dbversion'");
+    $hostname = $db->single("SELECT val FROM config WHERE cfg='hostname'");
+    $acc = $db->single("SELECT COUNT(1) FROM accounts");
+    $tr = $db->single("SELECT COUNT(1) FROM transactions");
+    $masternodes = $db->single("SELECT COUNT(1) FROM masternode");
+    $mempool = $db->single("SELECT COUNT(1) FROM mempool");
+    $peers = $db->single("SELECT COUNT(1) FROM peers WHERE blacklisted<UNIX_TIMESTAMP()");
+    api_echo([
+        'hostname'     => $hostname,
+        'version'      => VERSION,
+        'dbversion'    => $dbVersion,
+        'accounts'     => $acc,
+        'transactions' => $tr,
+        'mempool'      => $mempool,
+        'masternodes'  => $masternodes,
+        'peers'        => $peers
+    ]);
+} elseif ($q === 'checkAddress') {
+    /**
+     * @api            {get} /api.php?q=checkAddress  22. checkAddress
+     * @apiName        checkAddress
+     * @apiGroup       API
+     * @apiDescription Checks the validity of an address.
+     *
+     * @apiParam {string} account Account id / address
+     * @apiParam {string} [public_key] Public key
+     *
+     * @apiSuccess {boolean} data True if the address is valid, false otherwise.
+     */
+
+    $address=$data['account'];
```

Comment:
I have been reading the changes and I have been wondering: do you use some kind of checklist (in Intellij we call it [CheckStyle](https://github.com/HPI-Information-Systems/Metanome/wiki/Installing-the-google-styleguide-settings-in-intellij-and-eclipse)) for the formats of the code? 

Yours looks pretty clean but, not the same styling in every assignation. Nothing wrong with it, just wondering.